bundle/edu.gemini.seqexec.odb/src/main/scala/edu/gemini/seqexec/odb/SeqExecService.scala

```diff
@@ -83,25 +88,43 @@ object SeqExecService {
       }
     }
 
+  private def extractName(obs: ISPObservation): TrySeq[String] = trySeq {
+    obs.getDataObject match {
+      case o: SPObservation => o.getTitle.right
```

Comment:
`getTitle` is defined in the `ISPDataObject` from which `SPObservation` inherits, so you wouldn't need this match unless you just want to verify that the data object is in fact an `SPObservation`.  If it isn't though, nothing else will work anyway.

Reply:
Right, do you think I should just call it on the `DataObject`?

Reply:
Yeah, that's probably what I'd do.